### PR TITLE
[release-v1.5] SRVKE-1353 Multiarch test images

### PR DESCRIPTION
**Pull request by mgencur**
https://issues.redhat.com/browse/SRVKE-1353

* Include github action for building multiarch test images, similar to what was done for eventing-kafka-broker: [link](https://github.com/openshift-knative/eventing-kafka-broker/blob/main/.github/workflows/multiarch-build.yaml). This github action references the [shared one in openshift-knative/hack repo](https://github.com/openshift-knative/hack/blob/main/.github/workflows/multiarch-build.yaml)
* Allows specifying TEST_IMAGE_TAG and KO_FLAGS for the ko build. These params will be passed by the shared github action

The images will be built upon any commit on the release branch. And will be published to https://quay.io/repository/openshift-knative/eventing/<IMAGE_NAME>

### CODE REVIEW

**Comment by mgencur** [comment]
/hold
I want to merge it after merging the same PR for "main" and after enabling Github actions for this repo.

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Comment by mgencur** [comment]
/unhold
Gitub actions enabled for this repo.

**Pull request merged**
